tests/network/vmi_multus.go

```diff
@@ -91,6 +91,7 @@ const (
 )
```

Comment:
The next commits are about ptp and macvtap nads. This one is about  sriov right?
Can you add that info to the commit message.

Reply:
Or squash all 3 commits into this one; it makes a better argument for this PR, imo.

Reply:
By saying Multus tests I mean all the tests in vmi_multus.go file including SRIOV, 
But no problem I will elaborate about what Multus includes in the commit message.

Not about the PR structure, I separated it to commits so it will be easier to follow the changes.

Reply:
@alonSadan @maiqueb 
I need to do some other change, and create another commit for SRIOV test suite.
Before I do that you think its fine to squash all commits?